e2e-tests/cypress/support/appellant-submission-upload-application/confirmPlanningApplicationRejectedBecause.js

```diff
@@ -1,6 +1,6 @@
 module.exports = (errorMessage) => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/upload-application');
+  cy.url().should('include', '/appeal-householder-decision/upload-application');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks